package.json

```diff
@@ -32,7 +33,7 @@
   "devDependencies": {
     "eslint": "^6.7.2",
     "eslint-plugin-bpmn-io": "^0.10.0",
-    "mocha": "^6.2.2",
+    "mocha": "^10.2.0",
```

Comment:
Updating to the latest version was neccessary to correctly work with es-module syntax